programs/mango-v4/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(unused_variables)]
```

Comment:
curious, why was this needed?

Reply:
When compiled without `enable-gpl`, none of the instruction function arguments will end up being used - unfortunately I didn't find a way to conditionally enable this flag.